README.md

```diff
@@ -165,9 +166,10 @@ func (_m *SendFunc) Execute(data string) (int, error) {
 	return r0, r1
 }
 
-// NewSendFunc creates a new instance of SendFunc. It also registers a cleanup function to assert the mocks expectations.
+// NewSendFunc creates a new instance of SendFunc. It also registers the testing.TB interface on the mock and a cleanup function to assert the mocks expectations.
 func NewSendFunc(t testing.TB) *SendFunc {
 	mock := &SendFunc{}
+    mock.Mock.Test(t)
```

Comment:
Formatting issue here (tab vs space), could you correct the line before this?

Reply:
fixed!